src/components/modals/TextInputModal.tsx

```diff
@@ -152,7 +152,7 @@ export function TextInputModal(props: Props) {
           type="secondary"
         />
       )}
-      <ModalCloseArrow onPress={() => bridge.resolve(undefined)} />
+      <ModalFooter fadeOut onPress={() => bridge.resolve(undefined)} />
```

Comment:
The `fadeOut` looks really bad on this modal.

Reply:
Agreed - I have opted to wrap the `TextInputModal` in a `ModalScrollArea`, which has now been refactored to include a `ModalFooter`. Happy to revert this one, along with other non-list modals changed in this PR (eg the `QrCodeModal`) if using `ModalScrollArea` presents other issues.